cloudfoundry-client/src/test/java/org/cloudfoundry/uaa/clients/BatchDeleteClientsRequestTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2013-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.cloudfoundry.uaa.clients;
+
+import org.junit.Test;
+
+public final class BatchDeleteClientsRequestTest {
+
+    @Test(expected = IllegalStateException.class)
+    public void noClientIdentification() {
+        BatchDeleteClientsRequest.builder()
+            .build();
+    }
+
+    @Test
+    public void valid() {
+        BatchDeleteClientsRequest.builder()
+            .clientIdentification(ClientIdentification.builder()
+                .clientId("client-id")
```

Comment:
This should be "test-client-id", and is simpler anyway when I eliminate `ClientIdentification`.